DXRModules/DeusEx/Classes/DXRBingoCampaign.uc

```diff
@@ -193,6 +209,46 @@ function bool IsLateStart(int mission)
     return dxr.flags.settings.starting_map > mission * 10 + 5;
 }
 
+function HandleSpecialPerson(string bindname, string goal) {
+    local int oldSeed;
+    local ScriptedPawn pawn;
+    local PlayerDataItem data;
+
+    if (IsBoardGoal(goal)) return;
+
+    data = class'PlayerDataItem'.static.GiveItem(player());
+    oldSeed = SetGlobalSeed(dxr.dxInfo.MissionNumber @ dxr.flags.newgameplus_loops $ " HandleSpecialPerson " $ bindname);
```

Comment:
newgameplus_loops isn't needed here since the seed is already incremented each NG+ loop

```suggestion
    oldSeed = SetGlobalSeed(dxr.dxInfo.MissionNumber $ " HandleSpecialPerson " $ bindname);
```

Reply:
Are you saying that passing an identical string to `SetGlobalSeed()` will produce different results across loops?

Reply:
```
simulated function int SetGlobalSeed(coerce string name)
{
    local int oldseed;
    oldseed = dxr.SetSeed( dxr.seed + dxr.Crc(name) );
    dxr.rngraw();// advance the rng
    return oldseed;
}
```